python_tutorials/README.md

```diff
@@ -0,0 +1,11 @@
+# Links to books and tutorials on Python
+
+---
+
+## Full length textbooks and tutorials
+
+[https://docs.python-guide.org/](The Hitchhiker's guide to Python) -- includes information on installing Python and setting up virtual environments an **important** way to keep your projects seperated  
```

Comment:
you have your names and links backwards.  description goes inside the square brackets and links go in the parentheses.

like this: \[The hitchiker's guide to Python\](https://docs.python-guild.org/)

Reply:
also, in the name of the link, please include a dash followed by the website it links to: i.e. \[The Hitchhiker's guide to Python - python-guild.org\]